Dockerfile

```diff
@@ -69,10 +59,7 @@ RUN apt-get install -y libfreetype6-dev && \
     pip install nolearn && \
     pip install Theano && \
     pip install pybrain && \
-    # Base ATLAS
-    apt-get install -y libatlas-base-dev && \
-    cd /usr/local/src && git clone --depth 1 https://github.com/ztane/python-Levenshtein && \
-    cd python-Levenshtein && python setup.py install && \
+    pip install python-Levenshtein && \
```

Comment:
adding a smoke test for this package to ensure there is no regresssion would be helpful. It should also be pretty straightforward given you can write a test asserting on the levenshtein distance between two words is equal to some expected distance.